src/auth/config.rs

```diff
@@ -699,17 +699,17 @@ impl AuthModule for CustomAuth {
     fn is_valid(&self) -> Result<bool, String> {
         is_all("server", &self.server)?;
 
-        &self.meta.is_valid()?;
-        &self.mvt.is_valid()?;
-        &self.stats.is_valid()?;
-        &self.clone.is_valid()?;
-        &self.schema.is_valid()?;
-        &self.user.is_valid()?;
-        &self.feature.is_valid()?;
-        &self.style.is_valid()?;
-        &self.delta.is_valid()?;
-        &self.bounds.is_valid()?;
-        &self.osm.is_valid()?;
+        self.meta.is_valid()?;
```

Comment:
Curious, what was the clippy messaging here?

Reply:
When calling a function there isn't any reason to do it on the borrowed value